ppa/archive/forms.py

```diff
@@ -387,6 +390,36 @@ class SearchWithinWorkForm(forms.Form):
             '_align': 'left'
     }))
 
+    def as_json(self):
```

Comment:
not anymore, although I considered reviving it re: the discussion on how to send the form's `action=` to the frontend, since Django knows it but I can't get it into the markup without breaking the form in some way...